ipaserver/install/cainstance.py

```diff
@@ -2490,6 +2495,7 @@ def update_ipa_conf(ca_host=None):
     parser.set('global', 'enable_ra', 'True')
     parser.set('global', 'ra_plugin', 'dogtag')
     parser.set('global', 'dogtag_version', '10')
+    parser.set('global', 'startup_timeout', api.env.startup_timeout)
```

Comment:
During installation, this part is called as step 24 of the CA installation:
```
  [24/33]: updating IPA configuration
```
but there is a restart of pki before that one that does not benefit from the startup timeout setting at step 17:
```
  [17/33]: starting certificate server instance
```
If possible the update should be done earlier.

Reply:
Done